pkg/util/util.go

```diff
@@ -472,3 +472,31 @@ func UpsertInterimVolume(kubeClient kubernetes.Interface, podSpec core.PodSpec,
 	}
 	return AttachPVC(podSpec, volumes, volumeMounts), nil
 }
+
+func ResourceKindShortForm(kind string) string {
+	switch kind {
+	case apis.KindDeployment:
```

Comment:
For official types match the short names defined here: https://kubernetes.io/docs/reference/kubectl/overview/#resource-types